templates.dist/form/applicantform.php

```diff
@@ -68,7 +68,7 @@
 
 ?>
 	<p>
-	<h1>Ihre Kontaktdaten</h1>
+	<h1><?php esc_html_e('Your contact details', 'onoffice'); ?></h1>
```

Comment:
This is the wrong text domain. See #288.

Reply:
@jmaas-onoffice I fixed it. Please review again